_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -542,7 +554,42 @@ public function display_plugins() {
 	}
 
 	/**
-	 * Fetch Gravatar hashes for public A12s from wpcom and display them as a list.
+	 * Fetch anonymous data about A12s from wpcom: total count, number of countries, languages spoken.
+	 *
+	 * @since 7.4
+	 *
+	 * @return array $data
+	 */
+	private function fetch_a8c_data() {
+		$data = get_transient( 'a8c_data' );
+		if ( false === $data ) {
+			$data = json_decode(
+				wp_remote_retrieve_body(
+					wp_remote_get( 'https://public-api.wordpress.com/wpcom/v2/jetpack-about' )
+				)
```

Comment:
```suggestion
				),
				true
```

As is, it is returning an object, so it is failing the array check on L571. We need to set the assoc_array arg as true to get the array out.

Reply:
Fixed 👍